src/WpfMath/Atoms/ScriptsAtom.cs

```diff
@@ -81,7 +81,8 @@ protected override Box CreateBoxCore(TexEnvironment environment)
                 shiftUp = resultBox.Height - texFont.GetSupDrop(superscriptStyle.Style);
                 shiftDown = resultBox.Depth + texFont.GetSubDrop(subscriptStyle.Style);
             }
-            else if (this.BaseAtom is CharSymbol charSymbol)
+            else if (this.BaseAtom is CharSymbol charSymbol
+                     && charSymbol.IsSupportedByFont(texFont, style))
```

Comment:
Maybe it should be inside the scope? I could not figure out what the ```shiftUp``` and ```shiftDown``` variables should be equal in this case

Reply:
For now, I believe this is okay, because we literally don't know the `supdrop` parameter of the ordinary font. For TeX fonts, this is a parameter that's defined by font metrics.

I'll create a new umbrella issue about guessing these parameters for system fonts, and add this as a motivating example there.